auth/auth.go

```diff
@@ -59,7 +59,7 @@
 //
 // First create an Auth object:
 //
-//  myAuth := auth.New(tcclient.Credentials{ClientId: "myClientId", AccessToken: "myAccessToken"})
+//  myAuth := auth.New(&tcclient.Credentials{ClientId: "myClientId", AccessToken: "myAccessToken"})
```

Comment:
Subtle change here - the New method takes a pointer argument now (*tcclient.Credentials rather than tcclient.Credentials as it was before).

This is the only change in all the generated code (which is reflected in generated comments and generated code).